pep-0544.txt

```diff
@@ -949,6 +1048,44 @@ this in type checkers for non-protocol classes could be difficult. Finally, it
 will be very easy to add this later if needed.
 
 
+Prohibit explicit subclassing of protocols by non-protocols
+-----------------------------------------------------------
+
+This was rejected mainly for two reasons:
```

Comment:
There are additional reasons:

* Explicit subclassing makes it explicit that a class implements a particular protocol, making subtyping relationships easier to see.
* Type checkers can warn about missing protocol members or members with incompatible types more easily, without having to use hacks like dummy assignments.